docs/modules/transform-sync/how-it-works.md

```diff
@@ -0,0 +1,37 @@
+# How the Transform Synchronization Feature Module works
+
+## What is Transform
+
+There are two concepts of the location of a SpatialOS entity:
+
+* The _transform_. This contains a location, rotation, and physics tick and is used by worker-instances to run physics simulations or render entities for a client.
+
+* The _position_. This only contains a location. This is defined by the standard library component `Improbable.Position` and is used by the SpatialOS load balancer to determine which worker should be delegated authority over a given entity-component.
+
+<%(#Expandable title="Why do we need two locations?")%>
```

Comment:
this is confusing: we use location for both
1. the position
2. as the two components which both contain a location as well

we definitely should not name them the same